pkg/build/api/register.go

```diff
@@ -1,14 +1,17 @@
 package api
 
 import (
+	kapi "k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/unversioned"
 	"k8s.io/kubernetes/pkg/runtime"
+	"k8s.io/kubernetes/pkg/watch/versioned"
 )
 
-const GroupName = ""
-const FutureGroupName = "build.openshift.io"
+const LegacyGroupName = ""
+const GroupName = "build.openshift.io"
```

Comment:
yeah, currently only cluster admin can create resources in group... we will have to duplicate bootstrap policy for all grouped resources?